ironfish-rust/src/frost_utils/split_spender_key.rs

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use group::GroupEncoding;
+use ironfish_frost::frost::{
+    frost::keys::IdentifierList,
+    keys::{KeyPackage, PublicKeyPackage},
+    Identifier,
+};
+use ironfish_zkp::{constants::PROOF_GENERATION_KEY_GENERATOR, ProofGenerationKey};
+use jubjub::SubgroupPoint;
+use rand::thread_rng;
+use std::collections::HashMap;
+
+use crate::{IncomingViewKey, OutgoingViewKey, PublicAddress, SaplingKey, ViewKey};
+
+use super::split_secret::{split_secret, SecretShareConfig};
+
+pub struct SpenderKeyConfig {
+    pub ak: [u8; 32],
+    pub pgk: ProofGenerationKey,
+    pub vk: ViewKey,
+    pub ivk: IncomingViewKey,
+    pub ovk: OutgoingViewKey,
+    pub address: PublicAddress,
+    pub key_packages: HashMap<Identifier, KeyPackage>,
+    pub pubkeys: PublicKeyPackage,
+}
+
+pub fn split_spender_key(
+    coordinator_sapling_key: SaplingKey,
+    min_signers: u16,
+    max_signers: u16,
+    secret: Vec<u8>,
+) -> SpenderKeyConfig {
+    let secret_config = SecretShareConfig {
+        min_signers,
+        max_signers,
+        secret,
+    };
+
+    let mut rng = thread_rng();
+    let (key_packages, pubkeys) =
+        split_secret(&secret_config, IdentifierList::Default, &mut rng).unwrap();
```

Comment:
We are not going to be using the default identifier list, we are going to use Andreas implementation of Identifiers from `ironfish-frost`